### ISSUE TITLE
Integrated Terminal: Multiline Urls are not handled correctly

### ISSUE THREAD

**Issue by webdeb** [Microsoft/vscode#28746]
- VSCode Version:
Version 1.13.0 (1.13.0)
- OS Version:
OSX - Sierra
Steps to Reproduce:

1. Open vscode integrated terminal
2. Produce an URL which has a line-break inside the console (this for example could be the output of git push with the merge-request creation link, but you can simulate it by just typing an URL basically, has the same effect)
3. Try to click on it with CMD+Click

- Expected result:

Full URL is parsed correctly

- Actual result:

Only the part, which is on the 1. line is parsed as the URL

![image](https://user-images.githubusercontent.com/14992140/27153633-7d813a26-5152-11e7-8cda-b86ec1c072f3.png)

NOTE: In iTerm its working as expected

![bildschirmfoto 2017-06-14 um 22 41 35](https://user-images.githubusercontent.com/14992140/27153679-af4bb7fc-5152-11e7-8a0d-9ff31a96a068.png)

**Comment by Tyriar** [comment]
Upstream issue: https://github.com/sourcelair/xterm.js/issues/24

**Comment by Tyriar** [comment]
I just pushed in support for this, there are currently the following known issues:

1. The prompt may or may not be "linkified", this depends on how the linefeed was added to the terminal. For example up to browse history may not work
2. Printing a long link and then reducing the width of the terminal will cause the underline to go much larger than desired. This should be fixed when reflow is implemented https://github.com/Microsoft/vscode/issues/23688
